### PR TITLE
removed curator from zboxcli

### PR DESCRIPTION
**Pull request by Hitenjain14**
Removed curator functionality from zboxcli . 

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/2184
- blobber:
- gosdk: https://github.com/0chain/gosdk/pull/819
- system_test: https://github.com/0chain/system_test/pull/578
- zwalletcli:
- Other: ...

### CODE REVIEW

File: cmd/addcurator.go [hunk 1]
```diff
@@ -1,55 +1 @@
 package cmd
```
Comment (dabasov) [inline cmd/addcurator.go]: remove file

**Review by dabasov — APPROVED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | rm_curator  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | rm-curator  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | rm_curator  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | rm-curator  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by dabasov** [comment]
passed here https://github.com/0chain/0chain/actions/runs/4366536380/jobs/7682602847

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | master  |
| gosdk  | staging  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | rm_curator  |
| gosdk  | rm_curator  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | rm_curator  |
| gosdk  | rm_curator  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**